code/Bike.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>BikeMyLife</title>
+    <link rel="stylesheet" type="text/css" href="styleproject.css"/>
+
+    <!-- If you're using google fonts, that should be added here -->
+  </head>
+  <body>
+    <img src="Bikeflowtrail.jpg">
+    <div class="title-box">
+      <h1>Bike My Life</h1>
+    </div>
+
+  <div class="text-box">
+
+      <h2>Mountainbike</h2>
+  
+      <!--Beschreibung Biken-->
+      <p>Das Mountainbike ist wohl mein liebstes Velo. Ich liebe es, nach einem anstrengenden
+          Aufstieg die Abfahrt hinunter zu brettern und mich den technischen Herausforderungen
+          zu stellen. Immer wieder kommt man, wie im Leben in Situationen in denen man sich
+          überwinden muss. Von unten einen Blick werfend auf das passierte Hinderniss, ist 
+          es meist nicht mehr ganz so beeindruckend wie von oben. 
+          Manchmal soll es aber auch leichter gehen und FlowTrails sind perfekt dafür. 
+          Oft gibt es eine Bahn, mit der man hochfahren kann und dann den Trail wieder runter
+          ohne grössere technische Schwierigkeiten. Da kommt man echt in einen Flow. 
+          Das Herz klopft und der Smile wird mit jeder Kurve breiter. 
+      </p>    
+      <a href="https://ride.ch">Für Bikeinteressierte</a>
+  </div>
+  
+  <menu>
```

Comment:
Hi! Nice to meet you! 👋 I'm Estefanía and I'll be taking a look at your code today so that we can leave it functional and shiny.🙂